app/app/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="colorPrimary">@color/dark_peertube_dark_color</color>
+    <color name="colorPrimaryDark">#00574B</color>
+    <color name="colorAccent">@color/dark_soundcloud_primary_color</color>
+
+    <color name="dark_youtube_statusbar_color">#ff4336</color>
```

Comment:
There should probably be no reference to YouTube here (though this is not important at all ;-) )